content-usable/design.html

```diff
@@ -25,342 +24,335 @@ <h2>
       </p>
       <p>
-        Using familiar design patterns, terms and symbols is key to
-        helping users who struggle to remember new
-        symbols and design. Users need the following to be familiar:
+        Make it clear to users what things are and how to use them. This includes clearly indicating the purpose of: 
       </p>
       <ul>
-	    <li>
-		<strong>The look and interaction:</strong> Having controls, links and buttons look and behave like what they are, 
-		lets users know what it is and how to use it. As soon as users need to discover the design pattern, some users will be locked out. 
-		</li>
-        <li>
-          <strong>Symbols:</strong> Use familiar symbols. For example, people may look for a
-          question mark for help. If you use a different symbol, fewer people
-          will find help.
-        </li>
-        <li>
-          <strong>Text:</strong> Use familiar terms. For example, some people have a limited
-          vocabulary. Uncommon terms may cause them difficulty.
-        </li>
-		<li>
-          <strong>Element location:</strong> Place elements in expected locations. For example,
-          people may look for the search on the top right hand corner of a page.
-          If it is somewhere else, it will be hard to find.
-        </li>
-        <li><strong>Getting help:</strong> Always make it easy to find help.</li>
+      	<li>a site,</li>
+      	<li>section of a site, </li>
+      	<li>page, </li>
+      	<li>section of a page, and</li>
+      	<li>controls.</li>
       </ul>
       <p>
-        Personalization can be extremely useful for designers who want to offer
-        familiarity and flexibility. Familiarity helps users with cognitive and
-        learning disabilities but is often based on the needs of the individual
-        user. Personalization allows users to customize their interface, which
-        is important as what is familiar for one user may not be familiar to
-        another.
+        Use headers, labels and other signposts to help users know the purpose of the page, region, or control.
+      </p>
+      <p>Help users understand how to use controls and elements on each page. </p>
+      <p>
+      	Use familiar design patterns, terms and icons to help users who struggle to remember new designs. Ensure the look, location and interaction of controls and other elements are familiar and consistent across the site. 
+      </p>
+      <p>
+      	Show a clear relationship between controls and the content they effect to help users understand the effect of possible actions and reduce potential confusion.
       </p>
+  
     
       <section> 
         <h3>Pattern: Make the Purpose of Your Page Clear</h3>
           <section><h5 class="coga-5">User Need</h5>
```

Comment:
@ruoxiran  Aren't we missing h4? I know we need to keep avoid the lower headings as they are not clearly formatted. Or was that h4 that was bad so we skip it?

Reply:
When I merged, I did not modify the original heading, but this is not a problem. W3C repec.js will reset all headings and ids in HTML. If you inspect the html code of http://w3c.github.io/coga/content-usable/, you will find all headings and ids be reformatted automatically.